smalltalksrc/VMMaker/SpurPlanningCompactor.class.st

```diff
@@ -162,6 +163,23 @@ SpurPlanningCompactor >> compact [
 	self endCompaction
 ]
 
+{ #category : #'space management' }
+SpurPlanningCompactor >> configureSavedFirstFieldsSpaceStart: theStart limit: theLimit isOldSpace: isOldSpace [
+	"Use teden to hold the savedFirstFieldsSpace."
+
+	<inline: true>
+	| limit |
+	limit := theLimit.
+	savedFirstFieldSpaceMaxSlots ifNotNil: [
+		limit := limit min:
+			(theStart + (savedFirstFieldSpaceMaxSlots * manager bytesPerOop)) ].
+
+	savedFirstFieldsSpace
+		start: theStart;
+		limit: limit.
+	savedFirstFieldsSpaceNotInOldSpace := isOldSpace not
```

Comment:
Ufff, this name of variable makes it harder 🤯

Reply:
it was already like this :P I extracted repeated code, so at least now the bad name is in less places :)